packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -38,7 +38,8 @@ interface IBridge {
         // Processing fee for the relayer.
         uint256 fee;
         // gasLimit to invoke on the destination chain. If this value is zero, only destOwner can
-        // process the message.
+        // process the message. this value will not be respected on retries or
```

Comment:
I guess with the latest change it will actually mostly be respected when retrying (at least as gas > gas_limit, not as an exact limit)

Reply:
Should we phrase it as "This value will mostly be respected when retrying (gas > gas_limit) but not when processed by the owner" then?